cairo-hints/scarb-hints-lib/serialization.rs

```diff
@@ -192,8 +192,27 @@ fn parse_value(
             let string = value
                 .as_str()
                 .ok_or_else(|| "Expected string for Felt252".to_string())?;
+            let processed_string =
+                if !string.starts_with("0x") && !string.chars().all(|c| c.is_digit(10)) {
```

Comment:
Why is this? Could you add a comment on why is checking for al characters being `10`

Reply:
A felt252 can be:
- a number 
- a short string (up to 31 characters)

We are construction a `Felt252` though a string. So there are few cases: 
- The string is an hexadecimal -> `Felt252::from_hex`
- The string is a string representing a number -> `Felt252::from_dec_str`
- The string is a string of chars -> 1. converts string to hex. 2. `Felt252::from_hex`